package.json

```diff
@@ -141,17 +140,19 @@
 		"eslint-plugin-json": "^4.0.1",
 		"eslint-plugin-prettier": "^5.5.1",
 		"express": "^5.1.0",
+		"fast-glob": "^3.3.3",
```

Comment:
Any reason for using fast-glob? glob is already required by other dependencies, so you could use it, right?